windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -145,6 +145,8 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 $gem_installer = $SRC_ROOT + '\bin\gem_installer'
 $core_gems_rb = $SRC_ROOT + '\core_gems.rb'
 $plugin_gems_rb = $SRC_ROOT + '\plugin_gems.rb'
+# Pin ffi to version '1.14.1'. Remove the line when https://github.com/ffi/ffi/issues/868 is resolved.
```

Comment:
By `the line`, did you mean `the following line`?
Optionally, I would avoid repeating the actual version in the comment, so I'd even make this something like:
`Pin ffi version until https://github.com/ffi/ffi/issues/868 is resolved.`